README.md

```diff
@@ -99,6 +99,7 @@ Additional detailed documentation is included on the wiki on GitHub.
  * Removed Visual Studio project files, added VSCode PowerShell extensions launch file
  * Added xSPDatabaseAAG resource
  * Fixed bug with xSPWorkManagementServiceApp schema
+ * Added version as optional parameter for the feature resource to allow upgrading features to a specific version
```

Comment:
can you change "feature" to be "xSPFeteature" there so its clear the exact resource that it applies to:

    Added version as optional parameter for the xSPFeteature resource to allow upgrading features to a specific version